src/web/mjs/connectors/OremangaNet.mjs

```diff
@@ -0,0 +1,53 @@
+import OremangaNet from '../engine/OremangaNet.mjs';
+import Manga from '../engine/Manga.mjs'
+
+export default class Oremanga extends OremangaNet {
```

Comment:
I explained to you on the previous PR that you are extending a non existing class , and yet you do the same.

What are you trying to do?